### ISSUE TITLE
Fund lost due to strange reason

### ISSUE THREAD

**Issue by DSofter** [ethereum/mist#2377]
#### System information
``````
Version: `0.8.10`
OS & Version: OSX 10.12.5
Node type: `geth`
``````
I have posted this issue in https://github.com/ethereum/go-ethereum/issues/14538, but I thought it'd be good to post it here to raise more awareness.

Basically here's what happened:
1. I downloaded Mist on a new computer
2. Chose main net
3. Set up an account
4. Created a contract wallet (DDD) based on the main account
5. Imported an account via private key from Metamask (main net), which has 1.97 ETH balance
6. Sent 1.97 ETH from the account (step 5) to the contract wallet

Then the contract wallet showed up like this, ignore the `HELLO` contract wallet which I created later:
![image](https://cloud.githubusercontent.com/assets/1687829/26559831/e5e6b2e2-4466-11e7-82b3-446780d5be35.png)

If I click into the `DDD` contract wallet, it looks like this:
![image](https://cloud.githubusercontent.com/assets/1687829/26559851/20d9a454-4467-11e7-88f5-3888f5c17891.png)

So basically my funds are locked up and I cannot access it anymore. Any idea what happened? Thanks a ton!

**Comment by DSofter** [comment]
Found out why, see: https://github.com/ethereum/go-ethereum/issues/14538
